modules/sdk-coin-ton/src/lib/utils.ts

```diff
@@ -48,15 +48,27 @@ export class Utils implements BaseUtils {
     }
   }
 
-  async getAddressFromPublicKey(publicKey: string): Promise<string> {
+  async getAddressFromPublicKey(publicKey: string, bounceable = true): Promise<string> {
     const tonweb = new TonWeb(new TonWeb.HttpProvider(''));
     const WalletClass = tonweb.wallet.all['v4R2'];
     const wallet = new WalletClass(tonweb.provider, {
       publicKey: TonWeb.utils.hexToBytes(publicKey),
       wc: 0,
     });
     const address = await wallet.getAddress();
-    return address.toString(true, true, true);
+    return address.toString(true, true, bounceable);
+  }
+
+  getAddress(address: string, bounceable = true): string {
+    if (bounceable) {
```

Comment:
what if we simply do

`return new TonWeb.Address(address).toString(true, true, bounceable);`